docs/references/config/config.toml.md

```diff
@@ -1540,6 +1543,12 @@ to re-join consensus with the same state it has before crashing.
 Recovering nodes that "forget" the actions taken before crashing are faulty
 nodes that are likely to present Byzantine behavior (e.g., double signing).
 
+## Consensus timeouts
```

Comment:
This document is structured to contain the same sections as the `config.toml`. So by adding this section, we are splitting the consensus section, which no longer reflects the `config.toml` structure.

I suggest you remove this section marker, and move the sentence starting with "For a more detailed explanation" to each of the timeout fields.